src/test/java/io/cucumber/StepDefinitions.java

```diff
@@ -61,7 +63,7 @@ public void setUp(Scenario scenario) throws MalformedURLException {
         sauceUtils = new SauceUtils(sauceREST);
     }
 
-    @After
+    @After("not @api")
```

Comment:
If you really want to distinguish scenarios with and without using an API, using a cucumber tag on the feature file and filtering tests by the tag would make sense

Reply:
See comment above about spinning up VMs